src/dxvk/dxvk_compute.cpp

```diff
@@ -49,6 +49,11 @@ namespace dxvk {
   
   void DxvkComputePipeline::compilePipeline() {
     std::vector<VkDescriptorSetLayoutBinding> bindings;
+
+    if (Logger::logLevel() <= LogLevel::Debug) {
+      Logger::debug("Compiling compute pipeline..."); 
+      if(m_cs != nullptr) Logger::debug(str::format("  cs  : ", m_cs ->debugName()));
```

Comment:
that null pointer check is actually not necessary, compute pipelines cannot be created without a compute shader.